sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosContainer.java

```diff
@@ -450,16 +450,18 @@ public <T> CosmosPagedIterable<T> readAllItems(
 
     /**
      * Reads an item in the current container.
-     * <br/>
```

Comment:
why are we reverting these changes? 
@tangerinous didn't you update these as part of improving the API docs?

Reply:
This was removed accidentally because my changes were based on the repo state before @tangerinous's PR merge. Fixed it, thanks.

Reply:
@M-Lipin - thanks for fixing the merge issues, can you please also check my other comments on the Not Found exception. Let's verify that behavior on these APIs before updating the java docs for them, thanks!